Fixtures/Miscellaneous/Extensions/MySourceGenExtension/Package.swift

```diff
@@ -10,11 +10,23 @@ let package = Package(
         //     target: "MySourceGenExt"
         // )
     ],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-argument-parser.git", .upToNextMinor(from: "0.3.1")),
```

Comment:
I forgot that I still have an external package reference here.  I wanted an example that uses SwiftArgumentParser, which a real tool would, but will change this test fixture.

Reply:
maybe use a local dependency like "../Foo"

Reply:
That's what we usually do but I didn't want to vendor all of SwiftArgParser.  In this case it isn't actually to test that we can use the dependency (that will come in a later fixture, with an extension in a depenency package), but this was just to have it be "real" since pretty much every command line tool source generator written in Swift will probably use ArgParser.

So I'll remove this one though. but `../Foo` is a good idea for the upcoming extension-in-a-dependency fixture.

Reply:
another option is to have this Fixture run only when testing locally so CI does not do network access

Reply:
That's a good point.  How'd you normally do that — environment variables?

Reply:
yea that's one way

Reply:
Sounds good.  In this case it's not essential for this test, so there's no point spending the cycles even on local tests.  I will modify the test case.